crates/miden-lib/src/note/mod.rs

```diff
@@ -15,6 +15,7 @@ use utils::build_swap_tag;
 
 pub mod scripts;
 pub mod utils;
+pub mod well_known_note;
```

Comment:
Do we still need the `scripts` module to be public? (i.e., is there something there that we can't get from the `WellKnownNote` enum?